t/shdict.t

```diff
@@ -15,6 +15,7 @@ my $pwd = cwd();
 
 our $HttpConfig = <<_EOC_;
     lua_shared_dict dogs 1m;
+    lua_shared_dict cats 16k;
```

Comment:
A 16k zone might be a bit too small for our testing purposes. I'd add some tests for 100k at least ;)